javascript/code-challenges/challenges-01.test.js

```diff
@@ -97,6 +115,9 @@ This function should use forEach to populate your grocery list based on the stor
 ------------------------------------------------------------------------------------------------ */
 
 const createList = (availableItems) => {
+  // let newArr = [];
+  // availableItems.forEach(i =>
+  //   if(i.apples=== true); // I need help with this. I am confused on how to write if statement with a forEach loop. I might be over thinking it but im stuck on this.
```

Comment:
Couple things: you're accessing your object props incorrectly. "i" does not have a property of apples, it has a property of "name" with the value of "apples". So i.apples isn't a thing. i.name would return "apples" at index 0 of your array. Does that make sense?
Finally, javascript have this nifty thing where you can check true or false just by asking if it is. So you can say
```js
availableItems.forEach(item => {
  if(item.available) { // if item.available === false, than item is not available. So you can just say "if item.available" and if it's true than it evaluates to true, if false it evals to false.
    //push into new array.
  }
}
```